### DISCUSSION TITLE
Build on musl-based distro

### DISCUSSION THREAD

**Discussion by ahmadraniri1994** [CuarzoSoftware/Louvre#34] [Tested Hardware & Platforms]
Hello, are musl-based distros supported ? I'm trying to build it on alpine linux and it complains about missing header (`bits/types/FILE.h`). I also get warning about redirecting incorrect #include : 

```
warning: #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
```

**Comment by ehopperdietzel** [comment]
It's probably just an issue with includes, in that case, it should have an easy solution. I'll try it and see how to fix it when I have time.

**Comment by ehopperdietzel** [comment]
I replaced `<bits/types/FILE.h>` with `<stdio.h>`  and it should now compile on alpine, the `<poll.h>` warnings are still there but I'll fix it, please let me know if it works for you. 

The changes are in the **musl** branch.

**Comment by ahmadraniri1994** [comment]
Thanks, for the fix. It's compiled now.
Note : I found same <poll.h> warnings on SRM.
